Request subdomain: xdevman.is-a.dev

xdevman1 opened the pull request:
# Requirements
- [x] I have **read** and **understood** the [Terms of Service](https://is-a.dev/terms).
- [x] I understand my domain will be removed if I violate the [Terms of Service](https://is-a.dev/terms).
- [x] My file is in the `domains` directory and has the `.json` file extension.
- [x] My file's name is lowercased and alphanumeric.
- [x] My website is **reachable** and **completed**.
- [x] I have provided sufficient contact information in the `owner` key.

# Website Preview
[Portfolio](https://xdevman1.github.io/Portfolio/)

xdevman1 committed:
Create devman.json

xdevman1 committed:
Create xdevman.json

omardotdev commented:
your website seems to be inaccessible.. can you remove the `CNAME` file on your repo and then add it back when the pr is merged

xdevman1 commented:
Hello, i am sorry for the issue, i removed the cname file from my repo but my gh pages link still redirects me to ``` xdevman.is-a.dev``` but i created a demo view of my website which you can check
- https://portfolio32-ch6h.onrender.com/

omardotdev commented:
> Hello, i am sorry for the issue, i removed the cname file from my repo but my gh pages link still redirects me to ` xdevman.is-a.dev`

its actually not redirecting now you just have to wait a little bit for github pages to actually make the change lol

> but i created a demo view of my website which you can check  https://portfolio32-ch6h.onrender.com/

seems pretty good

xdevman1 commented:
Oh I see now, thanks anything else I can do?

omardotdev commented:
> Oh I see now, thanks anything else I can do?

i've taken a look at the commit and it seems like you've made one file with the name `devman.json` which has nothing but a plus sign in it, you should probably remove that and your pr should be good to merge

xdevman1 committed:
Delete domains/devman.json

xdevman1 committed:
Update xdevman.json

xdevman1 commented:
> i've taken a look at the commit and it seems like you've made one file with the name `devman.json` which has nothing but a plus sign in it, you should probably remove that and your pr should be good to merge
I updated the CNAME to `https://portfolio32-ch6h.onrender.com/` and deleted the devman.json file, Is the pr ready to merge now? Thanks

xdevman1 committed:
Update xdevman.json

omardotdev commented:
> I deleted the devman.json file, Is the pr ready to merge now? Thanks
yeah
